src/software/simulated_tests/simulated_test_fixture.py

```diff
@@ -543,6 +544,7 @@ def simulated_test_runner():
                 simulator_proto_unix_io,
                 blue_full_system_proto_unix_io,
                 yellow_full_system_proto_unix_io,
+                ProtoUnixIO(),
```

Comment:
Do we not support autoref for simulated tests? If not, then we should add a task to support it I think. It seems like a great way to validate our tests (e.g. there should be no fouls...)

Reply:
It may be difficult to support Autoref with simulated tests. The port is coded into a config file that Autoref expects us to connect to. This means that when we run simulated_tests in CI, we either need to make it `exclusive` to make sure no other simulated tests run and try to connect to that port (which will slow down our simulated test runtimes in CI) or figure out a way to run multiple autorefs with their own config while (this isn't straight-forward to do as their CLI is a bit limited)

Reply:
don't we already have the limitation of `exclusive` for pytests though?

Reply:
I feel like we should be moving towards removing those `exclusive` tags, I think those are intended to be a stopgap till we figure out a way to parallelize the gamecontroller with simulated tests better